python/sanlock.c

```diff
@@ -1301,7 +1322,7 @@ py_killpath(PyObject *self __unused, PyObject *args, PyObject *keywds)
         size_t arg_len;
 
         item = PyList_GetItem(argslist, i);
-        p = PyString_AsString(item);
+        p = PyUnicode_AsUTF8(item);
```

Comment:
How can this work in python 2 or 3? What type is item on python 2? python 3?

This change requires a separate patch, with good documentation on the possible input, and a test with some parameters.